src/Compilers/CSharp/Test/Syntax/Syntax/SerializationTests.cs

```diff
@@ -13,6 +13,8 @@
 using Roslyn.Test.Utilities;
 using Xunit;
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
@jasonmalinowski the presumption here is that we do not actual send NFWs for compiler unit tests.